packages/infra/lib/ihe-stack/ihe-gateway.ts

```diff
@@ -76,7 +75,7 @@ export function createIHEGateway(stack: Construct, props: IHEGatewayProps): void
     db,
     name: `${name}Inbound`,
     dnsSubdomain: "inbound",
-    httpPorts: [portInboundPD, portInboundDQ],
+    httpPorts: [config.inboundPorts.patientDiscovery, config.inboundPorts.documentQuery],
```

Comment:
why is inbound DR not a thing?

Reply:
both use the same channel, `XCA Inbound Interface`